bots/pr/src/main/java/org/openjdk/skara/bots/pr/ContributorCommand.java

```diff
@@ -87,7 +87,7 @@ private Optional<EmailAddress> parseUser(String user, PullRequest pr, CensusInst
         }
 
         if (contributor.fullName().isPresent()) {
-            return Optional.of(EmailAddress.from(contributor.fullName().get(), contributor.username() + "@openjdk.org"));
+            return Optional.of(EmailAddress.from(contributor.fullName().get(), contributor.username() + "@" + censusInstance.configuration.census().domain()));
```

Comment:
The `configuration` is a private field in class `CensusInstance`, so we should use the getter method `configuration()`. But it is curious that the building and testing of the `Pre-submit tests` succeeded.